devel/micropython/Portfile

```diff
@@ -58,6 +66,10 @@ if {${name} eq ${subport}} {
     post-patch {
         reinplace "s|-Werror||g" ${build.dir}/Makefile
         reinplace "s|PREFIX = /usr/local|PREFIX ?= /usr/local|g" ${build.dir}/Makefile
+        # https://trac.macports.org/ticket/70707
+        reinplace "s|CC = clang|CC = ${configure.cc}|" ${build.dir}/Makefile
+        reinplace "s|-Werror||g" ${worksrcpath}/mpy-cross/Makefile
+        reinplace "s|CC = clang|CC = ${configure.cc}|" ${worksrcpath}/mpy-cross/Makefile
```

Comment:
Are you sure it's necessary to patch mpy-cross stuff here? Patching only in the subport isn't enough?

Reply:
Yeah, it goes into that folder as well, and the build breaks.

Reply:
Then maybe it's better to put it outside the `if {${name} eq ${subport}} {` block so that it's executed when building either subports.

Reply:
If you think that is not likely to be accidentally broken later on, let’s do this way.

Reply:
I don't see how it can be worse than what you have now, it just saves two lines. My habit is if something is needed by all subports I put it outside.
Actually I think what you can also do is move all the patch lines outside to one place - won't break anything, just redundant in one or another case. Your choice.

Reply:
> Actually I think what you can also do is move all the patch lines outside to one place - won't break anything, just redundant in one or another case

That’s a good idea. Let me do this.

Reply:
@i0ntempest Could you please take a look, is this okay now or should I change something?

Reply:
LGTM, will merge after checks are done. Thanks for the PR!